README.md

```diff
@@ -0,0 +1,171 @@
+
+![amo release][amo_version] ![github release][github_version]
+
+[amo_version]: https://img.shields.io/amo/v/vimperator.svg
+[github_version]:  https://img.shields.io/github/release/vimperator/vimperator-labs.svg
+
+**Make Firefox/Thunderbird look and behave like Vim**
+
+> **To beginners**: Welcome to the Vimperator-labs!  
+> This README isn't written description and help for the software itself.
+> See "Home page", "Add-ons page" or `:help` command for details.
+
+Links             | URL
+------------------|-----------------------------------------------------------
+Home page         | http://www.vimperator.org/
+Add-ons page      | https://addons.mozilla.org/en-US/firefox/addon/vimperator/
+Github repository | https://github.com/vimperator/vimperator-labs/
+
+<!--
+To the contributor:
+What's New topic has two sub-topics that are "pre-release" and the latest release.
+You should add an entry to "pre-release" if you fix bugs, add or change features.
+Moreover, put Issues number (Pull requests number if not) in the head of entry.
+
+When update version, replace "pre-release" header with new version number and date,
+and drop the previous version sub-topic, finally, don't forget to put next
+"pre-release" header in the head of topic.
+
+Note: Issues or Pull requests numbers don't automatically make links to the
+repository, but just leave it as it is because reduce contributor's burden.
+-->
+What's New
+----------
+
+##### v3.xx.x (pre-release)
+- #690: create README.md
+
+##### v3.16.0 (2017-02-01)
+- #619: add option 'newtaburl' to set default URL in new tabs
+- #568: fix loading the tabs module
+- #614: update max version
+- #621: add help for statuslinevisiblity
+- #566: extend Makefile for signing
+- #597: fixes and refactors in hints module
+- #626: showtexthints option
+- #636: delete trailing whitespace
+- #637: cleanup SVG files
+- #644: add ssl by default to statusline
+- #648: fix 'make xpi'
+- #508: fix :jumps
+- #660: fix \<P\> by initializing nsITransferable before usage
+- #654: remove uriIsPrefix, unsupported
+
+Installation (Vimperator)
+-------------------------
+
+- Download add-on from AMO (signed)
+
+  > Note: If AMO version is previous than repo's, you can build own version
+  > and use it until AMO's update.
+
+  1. Enter [Add-ons page](https://addons.mozilla.org/en-US/firefox/addon/vimperator/).
+  2. Click `Continue to Download` button.
+  3. Click `Add to Firefox` button.
+
+- Download add-on from Github repository (unsigned)
+
+  > Note: Since version 48, Installing add-on to Firefox has required signing.
+  > Unsigned add-on is for Unbranded Builds. See [mozilla wiki][wiki_signing].
+
+  1. Enter [Releases page](https://github.com/vimperator/vimperator-labs/releases).
+  2. Click `vimperator-3.N.N.xpi` link under Downloads topic.
+
+- Build own version (and signing)
+  See [Build own version](#build-own-version) topic.
+
+[wiki_signing]: https://wiki.mozilla.org/Add-ons/Extension_Signing
+
+Build own version
```

Comment:
Hm, this might be a tad too much to include in the readme. The instructions are good (:+1:), but might fit better in the wiki. Opinion @gkatsev ?